lang/lua-coxpcall/Makefile

```diff
@@ -8,18 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lua-coxpcall
-PKG_VERSION:=1.15.0
+PKG_VERSION:=1_17_0
```

Comment:
this will break things like opkg versioncompare won't it?  can't we leave the version as a proper version and just fix the urls?

Reply:
Personally I don't like it. It means either needing to fix the source, the url, and the buildir manually or use two variables for the version.